src/styles/components/Header.scss

```diff
@@ -0,0 +1,9 @@
+.header {
+  padding: 6rem 0.5rem 0 10rem;
```

Comment:
I might remove this padding for now. It's pretty specific, and we'll have layout changes coming up that would cause this to change anyway. It's fine to be smushed in the top left for now 🙂